vuepress/docs/next/tutorials/getting-started/openshift-install-by-operator.md

```diff
@@ -14,10 +14,9 @@ Scenario 1 is similar to the Entando quickstart style of deployment which can be
 - Familiarity with the OpenShift console and operation
 
 ## Add the Entando Operator to the OperatorHub
-A cluster admin can add the 7.0 version of the Entando Operator into the local OperatorHub using the following command.
-```shell
-oc apply -f https://raw.githubusercontent.com/entando/entando-releases/v7.0.1/dist/ge-1-1-6/samples/openshift-catalog-source.yaml
-```
+A cluster admin can add the 7.1 version of the Entando Operator into the local OperatorHub using the following command.
+
+<EntandoCode>oc apply -f https://raw.githubusercontent.com/entando/entando-releases/{{$site.themeConfig.entando.fixpack.v71}}/dist/ge-1-1-6/samples/openshift-catalog-source.yaml</EntandoCode>
```

Comment:
FYI, this file is broken but I have an open question in #tools-team to ask for a fix.